index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<main>
+    <h1 id="title"> Survey Form </h1>
```

Comment:
- [ ]  Great job 😃at replying to the comments, but I see that you told the student to add a header tag..... without explaining what to do, so kindly 😊 try to explain to the student why should he/her changed this part because this clear to you, but not clear to him/her.💯